src/node/node_class.hpp

```diff
@@ -934,8 +934,8 @@ Napi::Value WrappedObject<ClassType>::ProxyHandler::get_own_property_descriptor_
     Napi::EscapableHandleScope scope(env);
 
     // Napi::Object target = info[0].As<Napi::Object>();
-    Napi::String key = info[1].As<Napi::String>();
-    std::string text = key;
+    // Napi::String key = info[1].As<Napi::String>();
```

Comment:
Why not remove this, previous and next line?

Reply:
Yeah I guess I can remove them all! I wondered if the existing one was commented out for a reason